lib/checkPullRequestLabels.js

```diff
@@ -149,3 +155,83 @@ module.exports.checkChangelogLabel = async function(context) {
   });
   await context.github.issues.createComment(stateCommentParams);
 };
+
+/**
+ * @param {String} username
+ */
+const isUserAllowedToRemoveCriticalLabel = function(username) {
+  return (
+    username === SERVER_JOBS_ADMIN ||
+    RELEASE_COORDINATORS.includes(username));
+}
+
+/**
+ * @param {import('probot').Context} context
+ */
+module.exports.checkCriticalLabel = async function(context) {
+  /**
+   * @type {import('probot').Octokit.IssuesGetLabelResponse} label
+   */
+  const label = context.payload.label;
+
+  if(label.name === CRITICAL_LABEL_NAME) {
+    // Check that user is allowed to remove the label.
+    const user = context.payload.sender;
+    if (!isUserAllowedToRemoveCriticalLabel(user.login)) {
+      const commentParams = context.issue({
+        body:
+          'Hi @' +
+          user.login +
+          ', only members of the release team /cc @oppia/release-coordinators ' +
+          'are allowed to remove critical labels. ' +
+          'I will be adding it back. Thanks!',
+      });
+      await context.github.issues.createComment(commentParams);
+
+      const addLabelParams = context.issue({
+        labels: [CRITICAL_LABEL_NAME],
+      });
+      await context.github.issues.addLabels(addLabelParams);
+
+    }
+  }
+}
+
+/**
+ * @param {import('probot').Octokit.IssuesGetLabelResponse} label
+ */
+const isPrLabel = function(label) {
+  return label.name.startsWith('PR') || PR_LABELS.includes(label.name);
+}
+/**
+ * @param {import('probot').Context} context
+ */
+module.exports.checkForIssueLabel = async function(context) {
+  /**
+   * @type {import('probot').Octokit.IssuesGetLabelResponse} label
+   */
+  const label = context.payload.label;
+
+  if (!isPrLabel(label)) {
+    // Issue label got added.
+    const user = context.payload.sender;
+    const link = 'here'.link(
+      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
+        'labeling-issues-and-pull-requests'
+    );
+    const commentBody =
+      'Hi @' + user.login + ', the ' + label.name + ' label should only be ' +
+      'used on issues, and I’m removing the label. You can learn more about ' +
```

Comment:
Just add specifically that refer to point x.

Reply:
@ankita240796 I'm not sure I understand what you mean here. The link takes them to the exact section on labels to be used on pull requests.

Reply:
Ah ok, sorry I missed that all points are relevant here.